ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/CommandTopic.java

```diff
@@ -117,7 +116,6 @@ public void start() {
           log.warn("Backup is out of sync with the current command topic. "
               + "Backups will not work until the previous command topic is "
               + "restored or all backup files are deleted.", e);
-          return restoreCommands;
```

Comment:
We need to make the above exception more specific. We shouldn't just ignore any exception. Ditto for the other place this is used. I know this isn't part of your patch, but might as well do this now.

Reply:
I changed it to only catch a KsqlException or KsqlServerException